treeherder/seta/update_job_priority.py

```diff
@@ -0,0 +1,202 @@
+'''This module is used to add new jobs to the job priority table.
+
+This will query the Treeherder runnable api based on the latest task ID from
+mozilla-inbound's TaskCluster index.
+
+Known bug:
+ * Only considering mozilla-inbound makes SETA act similarly in all repositories where it is
+   active. Right now, this works for integration repositories since they tend to have
+   the same set of jobs. Unfortunately, this is less than ideal if we want to make this
+   work for project repositories.
+'''
+import copy
+import datetime
+import json
+import logging
+import os
+import time
+
+from treeherder.config.settings import (SETA_HIGH_VALUE_PRIORITY,
+                                        SETA_HIGH_VALUE_TIMEOUT,
+                                        SETA_LOW_VALUE_PRIORITY,
+                                        SETA_LOW_VALUE_TIMEOUT)
+from treeherder.etl.seta import (parse_testtype,
+                                 valid_platform)
+from treeherder.seta.common import (job_priority_index,
+                                    unique_key)
+from treeherder.seta.models import JobPriority
+from treeherder.seta.runnable_jobs import RunnableJobsClient
+
+logger = logging.getLogger(__name__)
+
+
+def update_job_priority_table():
+    """Use it to update the job priority table with data from the runnable api."""
+    # XXX: We are assuming that the jobs accross 'mozilla-inbound', 'autoland' and 'fx-team'
+    #      are equivalent. This could cause issues in the future
+    data = query_sanitized_data(repo_name='mozilla-inbound')
+    if data:
+        return _update_table(data)
+    else:
```

Comment:
There are quite a few unnecessary `else`s throughout this PR - what are your thoughts on this code style? :-)